api/envoy/api/v2/cds.proto

```diff
@@ -417,6 +417,19 @@ message Cluster {
     // Deprecated settings from v1 config.
     // [#not-implemented-hide:] Hide from docs.
     DeprecatedV1 deprecated_v1 = 2 [deprecated = true];
+
+    // The hash function used to hash hosts onto the ketama ring.
```

Comment:
If we are only going to do this for Ketama, please put this inside RingHashLbConfig.

Reply:
This is already inside RingHashLbConfig unless I miss something.

Reply:
Sorry read the diff wrong, yes this part looks good.